tests/templates/kuttl/keycloak-user-info/10-install-opa.yaml.j2

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: test
+  labels:
+    opa.stackable.tech/bundle: "true"
+data:
+  test.rego: |
+    package test
+
+    userInfo(username) := http.send({"method": "POST", "url": "http://127.0.0.1:9476/user", "body": {"username": username}, "headers": {"Content-Type": "application/json"}, "raise_error": true}).body
+    currentUserInfo := userInfo(input.username)
```

Comment:
This should ideally be moved to some kind of "stackable library of rules", but we don't currently have a good way to deploy and manage that.

Reply:
Is this the subsequent work with bundle-builder we talked about?

Reply:
Yes.

Reply:
I've made a new issue to discuss this: https://github.com/stackabletech/opa-operator/issues/494